lessons/module-3/unit-testing-react.md

```diff
@@ -313,20 +298,40 @@ const Grocery = ({ name, quantity, notes, purchased, starred, onPurchase, onStar
 export default Grocery;
 ```
 
-Running the tests now, you should see a failure for the snapshot test, with the
-differences between the two snapshots in the console. Here is where you have to
-consider, 'am I ok with those changes?' If so, you can update the tests by
-typing `u` while `Jest` is still in watch mode. Alternatively, you can restart
-your test suite and run `npm test --updateSnapshot`.
+Run:
+
+```npm test```
+
+You should see a failure for the snapshot test, with the
+differences between the two snapshots in the console. Here is where
+you have to consider, 'am I ok with those changes?' 
+
+Since the changes are intentional, update the 
+snapshots in one of two ways: 
+
+- 1. Type `u` while `Jest` is still in watch mode. 
+- 2. Cancel the test suite, then run
+```npm test --updateSnapshot```
 
-### Testing for dynamic changes  
+Run:
 
-When you're testing your components, you're mainly testing presentation logic. Given our UI 
-will change based on application data (our component props), we'll want to make sure we 
-have tests for any conditional logic or dynamic changes. For example, our grocery component 
-changes visually based on whether or not the grocery item has been starred or purchased. 
+```npm test```
 
-We could start out with a simple test to see if it has the appropriate class if it's starred.
+### Testing Dynamic Changes 
+
+Above we said that `create-react-app` uses Jest as a test-runner [Jest](http://facebook.github.io/jest/) 
+(instead of Mocha). Jest also includes its own expectation library.  
+The syntax is notably similar to Chai's but there are differences. One 
+difference is `expect` syntax instead of `assert` syntax.  
```

Comment:
This is not correct. Chai uses both assert and expect.

Reply:
Changed it.  I hope it is better now.